src/lib/pages/library/ui/library-page.svelte

```diff
@@ -5,8 +5,10 @@
   import { UserContainer } from '$lib/entities/user';
   import { SignInButton, SignOutButton } from '$lib/features/authenticate';
   import type { Episode } from '$lib/shared/api';
+  import { getEpisodes } from '../lib/load-episodes';
 
-  $: episodes = Array.from<Episode>($likesStore.values());
+  let episodes: Episode[] = [];
+  $: getEpisodes([...$likesStore]).then((r) => (episodes = r));
```

Comment:
thought: I see you using this pattern quite frequently, I'd like to suggest a more conventional Svelte alternative — the [`{#await}` block](https://svelte.dev/tutorial/await-blocks), just in case you didn't know about it